model/scratch/dffml_model_scratch/logisticregression.py

```diff
@@ -0,0 +1,139 @@
+import pathlib
+from typing import AsyncIterator, Tuple, Any
+
+import numpy as np
+
+from dffml import (
+    config,
+    field,
+    entrypoint,
+    SimpleModel,
+    ModelNotTrained,
+    Accuracy,
+    Feature,
+    Features,
+    Sources,
+    Record,
+)
+
+
+@config
+class LogisticRegressionConfig:
+    predict: Feature = field("Label or the value to be predicted")
+    features: Features = field("Features to train on")
+    directory: pathlib.Path = field(
+        "Directory where state should be saved",
+        default=pathlib.Path("~", ".cache", "dffml", "scratch"),
+    )
+
+
+@entrypoint("scratchlgr")
+class LogisticRegression(SimpleModel):
```

Comment:
There needs to be some explanation in the docstring of what makes this different than `SLR`.